src/storm-dft/transformations/DftInstantiator.cpp

```diff
@@ -0,0 +1,125 @@
+#include "DftInstantiator.h"
+#include "storm/exceptions/NotImplementedException.h"
+#include "storm-dft/builder/DFTBuilder.h"
+
+
+namespace storm {
+    namespace transformations {
+        namespace dft {
+
+            template<typename ParametricType, typename ConstantType>
+            DftInstantiator<ParametricType, ConstantType>::DftInstantiator(storm::storage::DFT<ParametricType> const &dft) : dft(dft) {
+            }
+
+            template<typename ParametricType, typename ConstantType>
+            DftInstantiator<ParametricType, ConstantType>::~DftInstantiator() {
```

Comment:
I think destructors should always be made virtual?

Reply:
The destructor is virtual in the header file. I changed it to `= default` in the header file.